packages/webpack-cli/lib/groups/resolveConfig.js

```diff
@@ -172,7 +174,15 @@ const finalize = async (moduleObj, args) => {
 
             return rawConfig;
         }),
-    );
+    ).then((allConfigs) => {
```

Comment:
Avoid using `then`, we can use `async`/`await` safely, I think we need eslint rule for this